app/views/articles/show.html.erb

```diff
@@ -31,6 +31,36 @@
     </script>
   <% end %>
 
+<script type="application/ld+json">
+  {
+    "@context": "http://schema.org",
+    "@type": "Article",
+    "mainEntityOfPage": "True",
```

Comment:
according to https://schema.org/mainEntityOfPage this should be a property, correct?